specification/devopsinfrastructure/resource-manager/readme.md

```diff
@@ -91,7 +91,6 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-python
```

Comment:
context: Python SDK could be generated from Typespec directly so there is no need to keep generation configuration for swagger.